### ISSUE TITLE
Hide navigation toggle when there are no menu items

### ISSUE THREAD

**Issue by petermakowski** [canonical-web-and-design/maas-ui#4049]
**Expected behavior**
Main navigation menu toggle button should be hidden when there are no menu items.

**To Reproduce**
1. Go to the login page
2. Verify there are no menu items in the top level navigation
3. Resize the page to mobile breakpoint
4. The menu toggle is displayed even though there are no menu items

**Screenshots**
![Kapture 2022-06-09 at 09 23 45](https://user-images.githubusercontent.com/7452681/172789179-d095cb27-9bc7-43d5-b4da-a805a2e26dd5.gif)
